qiskit_nature/settings.py

```diff
@@ -14,67 +14,21 @@
 
 from __future__ import annotations
 
-import warnings
-
-
-class ListAuxOpsDeprecationWarning(DeprecationWarning):
-    """Deprecation Category for List-based aux. operators."""
-
-    pass
-
-
-class PauliSumOpDeprecationWarning(DeprecationWarning):
-    """Deprecation Category for PauliSumOp."""
-
-    pass
-
 
 class QiskitNatureSettings:
     """Global settings for Qiskit Nature."""
 
     def __init__(self) -> None:
         self._optimize_einsum = True
+        self._use_symmetry_reduced_integrals: bool = True
```

Comment:
Is there any reason to keep this now, along with the setter/getter? Its not used anywhere (and not covered by testing) and any user setting it will make no difference. If it was a question well we did not deprecate this or anything - even if a user did set this in their code the code would not fail after such a removal - it would just monkey patch that attribute here. Reading it would but then seems highly unlikely anyone would have use this and its a simply fix to anyones code had they done it.